<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Валидатор в работе:</string>
    </property>
    <widget class="QLineEdit" name="le_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;01-03, 13&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>01-89</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;02-16&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Mapper + Vector в работе</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Группа 2</string>
     </property>
     <widget class="QPushButton" name="pb_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sub</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>29</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Группа 1</string>
     </property>
     <widget class="QPushButton" name="pb_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sub</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>111</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>Группа 3</string>
     </property>
     <widget class="QPushButton" name="pb_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Sub</string>
      </property>
     </widget>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="Display">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
